openshift-container-platform-4/policies/AC-Access_Control/component.yaml

```diff
@@ -78,7 +78,11 @@
         party software such as an Identity Provider (IdP),
         Active Directory (MSAD), LDAP, etc.
 
-        To configure an IdP, see https://docs.openshift.com/container-platform/latest/authentication/identity_providers/configuring-ldap-identity-provider.html#configuring-ldap-identity-provider
+        To configure an IdP, see https://docs.openshift.com/container-platform/latest/authentication/understanding-identity-provider.html
+
+        Note that the using the htpasswd IDP type cannot cover this
+        control, therefore using httpasswd based IDPs should render
+        the system non-compliant.
```

Comment:
@JAORMX I think I remember we used to say in the original control assessment that the IDPs used should be either LDAP or RequestHeader, but I'm struggling to remember what would be wrong with the other IDP types. Do you remember maybe?

Reply:
No, I don't remember. I suggest we just check for the ones we know are not allowed. It'll then be up to the deployer to verify that their IdP is FedRAMP compliant.

Reply:
Right, only htpasswd seems to be non-compliant, so I'll go ahead and update the existing IDP rule with references to the AC-2(XXX) controls that require such IDP.